packages/cli/src/lib/commands/collect/options/openReport.ts

```diff
@@ -1,19 +1,15 @@
-import { argv } from 'yargs';
-import { Param } from './openReport.model';
-import { ArgvOption } from '../../../core/yargs/types';
+import { argv, Options } from 'yargs';
 import { getEnvPreset } from '../../../pre-set';
 
-export const param: Param = {
-  openReport: {
-    alias: 'e',
-    type: 'boolean',
-    description: 'Opens browser automatically after the user-flow is collected. (true by default)',
-    default: getEnvPreset().openReport as boolean,
-    requiresArg: true
-  }
-};
+export const openReport = {
+  alias: 'e',
+  type: 'boolean',
+  description: 'Opens browser automatically after the user-flow is collected. (true by default)',
+  default: getEnvPreset().openReport as boolean,
```

Comment:
This needs to be completly removed. It is a mistake to have getEnvPreset().openReport at all so yeah i am not fixing that here because i want it completly gone.